plugins/chat/spec/system/chat_composer_spec.rb

```diff
@@ -240,15 +240,11 @@
 
       file_path = file_from_fixtures("logo.png", "images").path
       cdp.with_slow_upload do
-        attach_file(file_path) do
-          channel_page.open_action_menu
-          channel_page.click_action_button("chat-upload-btn")
-        end
-
+        attach_file("channel-file-uploader", file_path, make_visible: true)
```

Comment:
not sure why but the manual way is not working at all anymore, I tried various variations but nothing worked, this is probably faster anyways, it's not 100% perfect because we don't actually test that the click is working, but most tests don't have this goal anyways